Sources/PIRProcessDatabase/ProcessDatabase.swift

```diff
@@ -311,13 +311,18 @@ struct ProcessDatabase: ParsableCommand {
             """)
     var configFile: String
 
+    @Flag(name: .customLong("parallel"),
+          inversion: .prefixedNo,
+          help: "Enables parallel processing.")
+    var parallel = false
```

Comment:
I am wondering, if the default should be `true` for this.

@fboemer please also take a look at this PR.

Reply:
Default `true` sounds fine to me.
Rest of the PR LGTM as well.

Reply:
Changed it to `true`